smart-contracts/scripts/all_networks

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
```

Comment:
need to rebase after merging https://github.com/unlock-protocol/unlock/pull/11045

Reply:
Good, approved that other one so we can work on this here!

Reply:
ok done